fastlane/Fastfile

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
2 questions!

1. Does this file need to be nested like this? Woujld be easier if it's just root level.

2. Can we add to the README maybe under the `## Testing` instructions on how to run the unit tests via command line:
```
bundle install
bundle exec fastlane tests
```

Reply:
Yes! Good call. I'll update the readme. 

Re: nesting fastlane -  by default, `fastlane init` nests the file this way. I'm not sure if this is important to how fastlane operates, but any reporting files that get created as part of the fastlane build process would also be output here.

Reply:
I kept the existing file structure, since there are other output files that are generated through running our actions (testing now, but also thinking to when we add other actions that have their own output files). 

Updated the readme as well.

Reply:
Also, after comparing the performance of `run_tests` and the existing xcodebuild command, I opted to keep the build command. This also let me remove the `run` executable ability from our test target.